src/components/TimelineNav/TimelineNav.tsx

```diff
@@ -389,16 +371,14 @@ export const TimelineNav = ({
         <ol
           className={clsx(
             styles['timeline-nav__list'],
+            styles['timeline-nav__list--ordered'],
             isActive && styles['eds-is-active'],
-            {
-              [styles['timeline-nav__list--ordered']]: variant === 'ordered',
-            },
           )}
           role="tablist"
         >
           {timelineNavItems().map((tab, i) => {
             const isActive = activeIndexState === i;
-            const itemVariant = variant && tab.props.variant;
+            const itemVariant = tab.props.variant;
```

Comment:
@jinlee93 the chromatic test got to this line, which now will always have a value. previously, `variant` being undefined would set `itemVariant` to undefined i think

Reply:
That does make sense